run/browsers_test.py

```diff
@@ -28,28 +28,37 @@
     'os_version': None,
 }
 
+
 class TestBrowsers(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        filepath = os.path.join(os.path.dirname(__file__), os.pardir, 'browsers.json')
+        filepath = os.path.join(os.path.dirname(__file__),
+                                os.pardir, 'browsers.json')
         with open(filepath) as f:
             cls.browsers = json.load(f)
 
     def test_all_platforms_have_only_valid_characters(self):
         for platform_id, platform_info in self.browsers.items():
-            self.assertTrue(re.match(VALID_PLATFORM_ID_REGEX, platform_id),
-                'platform_id with invalid characters: %s' % platform_id)
+            self.assertTrue(
+                re.match(VALID_PLATFORM_ID_REGEX, platform_id),
+                'platform_id with invalid characters: %s' % platform_id
+            )
 
     def test_all_browsers_have_required_fields(self):
         for platform_id, platform_info in self.browsers.items():
             for key, valid_values in REQUIRED_PLATFORM_FIELDS.items():
-                self.assertTrue(key in platform_info.keys(),
-                    'Required field missing: %s (platform %s)' % (key, platform_id))
+                self.assertTrue(
+                    key in platform_info.keys(),
+                    'Required field missing: %s (platform %s)' %
+                    (key, platform_id)
+                )
 
                 if REQUIRED_PLATFORM_FIELDS[key] is not None:
-                    self.assertTrue(platform_info[key] in REQUIRED_PLATFORM_FIELDS[key],
-                        'Field has invalid value: %s (platform %s)' % (key, platform_id))
+                    self.assertTrue(
+                        platform_info[key] in REQUIRED_PLATFORM_FIELDS[key],
+                        'Field has invalid value: %s (platform %s)'
+                        % (key, platform_id))
```

Comment:
I don't have an opinion (which might be why I was inconsistent in this PR).

I'll go with keeping the closing parenthesis on the previous line since that looks a little more pythonic. I'll update the PR.